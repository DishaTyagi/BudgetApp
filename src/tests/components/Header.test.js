import React from 'react';
import Header from '../../components/Header';
import ReactShallowRenderer from 'react-test-renderer/shallow';

test('render header component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);      //arg contains the jsx we're trying to render here.
    expect(renderer.getRenderOutput()).toMatchSnapshot();    //gives the output of the jsx we are rendering. here Header.
});