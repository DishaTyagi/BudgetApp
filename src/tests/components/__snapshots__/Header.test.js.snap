// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render header component correctly 1`] = `
<header>
  <h1>
    expensify
  </h1>
  jkl
  <Unknown
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </Unknown>
  <Unknown
    activeClassName="is-active"
    to="/help"
  >
    Help
  </Unknown>
</header>
`;
